### DISCUSSION TITLE
Karpenter Fargate Support

### DISCUSSION THREAD

**Discussion by larskinder** [aws-ia/terraform-aws-eks-blueprints-addons#422] [Q&A]
Hi,

I am currently trying to leverage the addons and noticed that they don't create a Fargate profile for Karpenter.
To me it seems like a convenient feature to run the Karpenter pods on Fargate. 

To overcome this, I added the Fargate profile for Karpenter directly in my EKS module. 

Is this the default way or have I missed something?

Thanks in advance.

**Comment by bryantbiggs** [comment]
No - that is the correct assumption. The EKS module creates the cluster and compute resources, the addons module simply deploys resources onto an existing cluster
